eng/tools/gen-sdk/cmd/gen-sdk.js

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
```

Comment:
I'm not sure "gen-sdk" is the best name here as it likely will be confused with "spec-gen-sdk". We should probably come up with a name that better illustrates what it is doing.

Reply:
Maybe call it "run-spec-gen-sdk-for-set" or something along those lines.

Reply:
How about "run-spec-gen-sdk" since it will support running for multiple cases, such as single spec config, a set of spec config.

Reply:
Yes that is fine. Another potential option would be `spec-gen-sdk-runner` but I leave it to you.